sql/sqlspec/migratespec.go

```diff
@@ -0,0 +1,31 @@
+package sqlspec
+
+import (
+	"ariga.io/atlas/schema/schemaspec"
+)
+
+type (
+	// Change is the interface implemented by change specifications. Change instances are supposed
+	// to be mappable to schema.Change instances.
+	Change interface {
+		c()
```

Comment:
https://github.com/ariga/atlas/blob/master/sql/schema/migrate.go#L24
```suggestion
		change()
```